editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -1920,6 +1918,8 @@ void TileSetAtlasSourceEditor::_tile_alternatives_control_gui_input(const Ref<In
 				drag_type = DRAG_TYPE_MAY_POPUP_MENU;
 				drag_start_mouse_pos = alternative_tiles_control->get_local_mouse_position();
 			} else if (drag_type == DRAG_TYPE_MAY_POPUP_MENU) {
+				drag_type = DRAG_TYPE_NONE;
```

Comment:
I think we should set `drag_type = DRAG_TYPE_NONE` even if `drag_type != DRAG_TYPE_MAY_POPUP_MENU`. As a safety measure.
Thus the logic should likely be changed to:
```
if (mb->is_pressed()) {
   ...
} else {
   if (drag_type == DRAG_TYPE_MAY_POPUP_MENU) {
      ...
   }
   drag_type = DRAG_TYPE_NONE;
}
```

That will make it 100% sure that dragging is "released" when mouse is released.